hazelcast/src/test/java/com/hazelcast/spi/discovery/DiscoverySpiTest.java

```diff
@@ -252,6 +252,11 @@ private static void setEnvironment(String key, String value) throws Exception {
                 map.put(key, value);
             }
         }
+
+        Field field = Class.forName("java.lang.ProcessEnvironment").getDeclaredField("theCaseInsensitiveEnvironment");
```

Comment:
Hi @noctarius ,
What is the whole purpose of `setEnviroment` method? Can you please shed some light? cc: @kobalski